File: typescript/base-hmi-types/src/screens/base/HmiDcsFaceplateContainer.ts
import { HmiWindowBase } from "./HmiWindowBase.js";
import { HmiObjectType } from "./HmiObjectType.js";

export class HmiDcsFaceplateContainer extends HmiWindowBase {
  constructor() {
    super();
    this.hmiObjectType = HmiObjectType.HmiDcsFaceplateContainer;
  }
}

File: src/BaseHmiTypes/Screens/Base/HmiDcsFaceplateContainer.cs
namespace BaseHmiTypes.Screens.Base;

public class HmiDcsFaceplateContainer : HmiPaintedScreenItemBase
{
    public HmiDcsFaceplateContainer()
    {
        HmiObjectType = BaseHmiTypes.Screens.Base.HmiObjectType.HmiDcsFaceplateContainer;
    }

    public string FaceplateName { get; set; }

    public string FaceplateVersion { get; set; }

}
